railties/lib/rails/commands/test/test_command.rb

```diff
@@ -32,6 +32,31 @@ def perform(*)
         Rails::TestUnit::Runner.parse_options(args)
         Rails::TestUnit::Runner.run(args)
       end
+
+      # Define Thor tasks to avoid going through Rake and booting twice when using bin/rails test:*
+
+      Rails::TestUnit::Runner::TEST_FOLDERS.each do |name|
+        define_method(name) do
```

Comment:
```suggestion
        define_method(name) do
```
```suggestion
        define_method(name) do |*|
```

Otherwise any extra arguments/flags will be raising an error. E.g.:

```
rails test:controllers -b                                                                                                                          
Traceback (most recent call last):                                                                                                                 
        9: from bin/rails:6:in `<main>'                                 
        8: from /home/artem/.rbenv/versions/2.7.5/lib/ruby/gems/2.7.0/gems/bootsnap-1.11.1/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:30:in `require'
        7: from /home/artem/.rbenv/versions/2.7.5/lib/ruby/gems/2.7.0/gems/bootsnap-1.11.1/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:30:in `require'
        6: from /home/artem/.rbenv/versions/2.7.5/lib/ruby/gems/2.7.0/gems/railties-7.0.2.3/lib/rails/commands.rb:18:in `<main>'
        5: from /home/artem/.rbenv/versions/2.7.5/lib/ruby/gems/2.7.0/gems/railties-7.0.2.3/lib/rails/command.rb:48:in `invoke'
        4: from /home/artem/.rbenv/versions/2.7.5/lib/ruby/gems/2.7.0/gems/railties-7.0.2.3/lib/rails/command/base.rb:87:in `perform'
        3: from /home/artem/.rbenv/versions/2.7.5/lib/ruby/gems/2.7.0/gems/thor-1.2.1/lib/thor.rb:392:in `dispatch'
        2: from /home/artem/.rbenv/versions/2.7.5/lib/ruby/gems/2.7.0/gems/thor-1.2.1/lib/thor/invocation.rb:127:in `invoke_command'                                                                                                                                                                  
        1: from /home/artem/.rbenv/versions/2.7.5/lib/ruby/gems/2.7.0/gems/thor-1.2.1/lib/thor/command.rb:27:in `run'
/home/artem/projects/slp2_chilli/lib/rails/command/test_command.rb:26:in `block (2 levels) in <class:TestCommand>': wrong number of arguments (given 1, expected 0) (ArgumentError)
```